ebl/v3/issuance/EBL_ISS_v3.0.0.yaml

```diff
@@ -609,6 +609,10 @@ components:
             description: |
               A line of the address to be displayed on the transport document.
             example: 'Strawinskylaan 4117'
+          example:
+            - Strawinskylaan 457
+            - 1077 XX Amsterdam
+            - Nederlandene
```

Comment:
Is it Nederlanden?

Reply:
No idea - I copied it from Google...
<img width="524" alt="image" src="https://github.com/user-attachments/assets/8a92c73d-8d8f-481e-a6ac-9c8b3f303a15">